### DISCUSSION TITLE
Network Usage compared to Vienna Ensemble Pro

### DISCUSSION THREAD

**Discussion by sparksandshadows** [apohl79/audiogridder#578] [General]
I'm just curious: why does Audiogridder have a much higher network usage than vienna ensemble pro?
![image](https://user-images.githubusercontent.com/82538597/119204702-fe327d00-ba4a-11eb-8e47-b52f4e3421a1.png)
I have 57 instances of agridder in the template, versus over a thousand stereo returns from VEP (not playing at the moment).

Not a dealbreaker, and agridder is obviously a different beast--just curious.

**Comment by apohl79** [comment]
Hard to say. The design an network protocols and inner workings are different. It might be due to the remote control capabilities of AG or the kind of plugins loaded (inst/fx). I don't know how VEP works.

**Comment by jaacMusic** [comment]
Just to add a bit more to this. I found out that the network consumption for AG was on par with Waves' SoundGrid Servers.
Also, it would be interesting to see the network load when using VEP.

**Reply by sparksandshadows** [reply]
That is interesting--I will say this may say more about VEP than Audiogridder. Once you start receiving audio through VEP it pretty soon eclipses agridder in terms of network usage, whereas agridder seems to not fluctuate.
